src/hooks/dx11.rs

```diff
@@ -169,10 +168,11 @@ impl ImguiRenderer {
         trace!("Initializing renderer");
 
         let dev: ID3D11Device = swap_chain.GetDevice().expect("GetDevice");
-        let mut dev_ctx: Option<ID3D11DeviceContext> = None;
-        dev.GetImmediateContext(&mut dev_ctx);
-        let dev_ctx = dev_ctx.unwrap();
-        let sd = swap_chain.GetDesc().expect("GetDesc");
+        let dev_ctx: ID3D11DeviceContext = dev.GetImmediateContext().unwrap();
+
+        let dev_ctx = dev_ctx;
+        let mut sd = DXGI_SWAP_CHAIN_DESC::default();
+        swap_chain.GetDesc(&mut sd).expect("GetDesc");
```

Comment:
Before we had to pass `&mut dev_ctx` as out parameter and get `sd` by return, now we have to do the exact opposite and get `dev_ctx` as return value and pass `&mut sd` as out parameter. I can't find an explanation as to why this is the case.

Of course the absolutely obvious, consistent and ideal thing that we should be able to do here would be:

```
let dev_ctx = dev.GetImmediateContext()?;
let sd = swap_chain.GetDesc()?;
```

I have no idea what this change entails, besides moving the code from "not compiles" to "compiles", but I can't exclude these things have side effects that lead to crashes.